docs/release/3.4/release-3-4-5.md

```diff
@@ -0,0 +1,117 @@
+OSG Software Release 3.4.5
+==========================
+
+**Release Date**: 2017-11-14
+
+Summary of changes
+------------------
+
+This release contains:
+
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.5%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+!!! note
+    OSG 3.4 contains only 64-bit components.
+
+!!! note
+    StashCache is supported on EL7 only.
+
+!!! note
+    xrootd-lcmaps will remain at 1.2.1-2 on EL6.
+
+Detailed changes are below. All of the documentation can be found [here](../../).
+
+Known Issues
+------------
+
+-   Because the Gratia system has been turned off, RSV emits the following warning: `WARNING: Gratia server gratia-osg-prod.opensciencegrid.org:80 not responding to obtain last contact details`. This warning can safely be ignored.
+-   In GlideinWMS, a small configuration change must be added to account for changes in HTCondor 8.6. Add the following line to the HTCondor configuration.
+
+        COLLECTOR.USE_SHARED_PORT=False
+
+Updating to the new release
+---------------------------
+
+To update to the OSG 3.4 series, please consult the page on [updating between release series](../release_series).
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
+
+### Update Software
+
+Once the new repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note
+Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
```

Comment:
This and the next `!!! note` formatting is wrong (subsequent lines need to be preceded by 4 spaces), see previous `!!! note`s